src/features/search/componets/SearchBar/SearchBar.tsx

```
@@ -57,48 +74,111 @@
+                    title:
+                      '✨' + i18n.t('search.mcda_create_analysis') + ` "${item.name}"`,
+                    value: item.name,
+                    hasDivider: true,
+                  }}
+                  highlighted={highlightedIndex === index}
+                  itemProps={{
+                    onClick: () => handleItemSelect(item),
+                    role: 'option',
+                  }}
+                ></SelectItem>
+              );
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)